src/storybook/config.js

```diff
@@ -1,151 +0,0 @@
-  require("./stories/customers/CustomerListPage");
-
-  // Discounts
-  require("./stories/discounts/DiscountCountrySelectDialog");
-  require("./stories/discounts/SaleCreatePage");
-  require("./stories/discounts/SaleDetailsPage");
-  require("./stories/discounts/SaleListPage");
-  require("./stories/discounts/VoucherCreatePage");
-  require("./stories/discounts/VoucherDetailsPage");
-  require("./stories/discounts/VoucherListPage");
-
-  // Homepage
-  require("./stories/home/HomePage");
-
-  // Navigation
-  require("./stories/navigation/MenuCreateDialog");
-  require("./stories/navigation/MenuDetailsPage");
-  require("./stories/navigation/MenuItemDialog");
-  require("./stories/navigation/MenuListPage");
-
-  // Pages
-  require("./stories/pages/PageDetailsPage");
-  require("./stories/pages/PageListPage");
-
-  // Plugins
-  require("./stories/plugins/PluginDetailsPage");
-  require("./stories/plugins/PluginsListPage");
-
-  // Products
-  require("./stories/products/ProductCreatePage");
-  require("./stories/products/ProductImagePage");
-  require("./stories/products/ProductListPage");
-  require("./stories/products/ProductUpdatePage");
-  require("./stories/products/ProductVariantCreatePage");
-  require("./stories/products/ProductVariantImageSelectDialog");
-  require("./stories/products/ProductVariantPage");
-
-  // Orders
-  require("./stories/orders/OrderBulkCancelDialog");
-  require("./stories/orders/OrderCancelDialog");
-  require("./stories/orders/OrderCustomer");
-  require("./stories/orders/OrderDetailsPage");
-  require("./stories/orders/OrderDraftCancelDialog");
-  require("./stories/orders/OrderDraftListPage");
-  require("./stories/orders/OrderDraftPage/OrderDraftPage");
-  require("./stories/orders/OrderFulfillmentCancelDialog");
-  require("./stories/orders/OrderFulfillmentTrackingDialog");
-  require("./stories/orders/OrderHistory");
-  require("./stories/orders/OrderListPage");
-  require("./stories/orders/OrderMarkAsPaidDialog");
-  require("./stories/orders/OrderPaymentDialog");
-  require("./stories/orders/OrderPaymentVoidDialog");
-  require("./stories/orders/OrderProductAddDialog");
-  require("./stories/orders/OrderShippingMethodEditDialog");
-  require("./stories/orders/OrderInvoiceList");
-
-  // Product types
-  require("./stories/productTypes/ProductTypeCreatePage");
-  require("./stories/productTypes/ProductTypeDetailsPage");
-  require("./stories/productTypes/ProductTypeListPage");
-
-  // Shipping
-  require("./stories/shipping/ShippingZoneCountriesAssignDialog");
-  require("./stories/shipping/ShippingZoneCreatePage");
-  require("./stories/shipping/ShippingZoneDetailsPage");
-  require("./stories/shipping/ShippingZonesListPage");
-
-  // Site settings
-  require("./stories/siteSettings/SiteSettingsPage");
-
-  // Translations
-  require("./stories/translations/TranslationsEntitiesListPage");
-  require("./stories/translations/TranslationsLanguageListPage");
-}
-
-configure(loadStories, module);
```

Comment:
It not just removes stories snapshots, it removes stories from storybook, am I right?